src/components/BSelect/BSelect.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <div>
+      <label>{{selectLabel}}<span v-if=isRequired>*</span></label>
+    </div>
+    <div>
+      <select
+        v-bind:aria-label="selectLabel"
+        v-bind:required="isRequired ? true : false"
+      >
+        <option disabled selected>Choose Option</option>
+        <option
+          v-for="option in options"
+          v-bind:key="option.id"
+          v-bind:option="option"
```

Comment:
Should this be "option" or "option.text"?

Reply:
Actually, is having that necessary or is it a holdover from a previous version? Since the id is saved as the key and value and the text is displayed, that's all each option is.